tests/add_user.spec.ts

```diff
@@ -16,18 +17,24 @@ test.describe("Add User functional", () => {
     const addUserSteps = new AddUserSteps(page);
     const generator = new Generator();
 
+    const gender = generator.getGender();
```

Comment:
move Generator constant declaration out of tests, create it just once for the test file